week-06/Webshop/Webshop/Models/StockList.cs

```diff
@@ -45,19 +38,19 @@ public static List<Item> GetSearch(List<Item> items, string query)
             return items.Where(n => n.Description.ToLower().Contains(query.ToLower()) || n.Name.ToLower().Contains(query.ToLower())).ToList();
         }
 
-        public static List<Item>filterByPrice (string bounds, double amount)
+        public static List<Item>filterByPrice (List<Item> items, string bounds, double amount)
         {
             var priceFiltered = new List<Item>();
             switch (bounds)
             {
                 case "above":
-                    priceFiltered = StockList.Inventory.Where(n => n.Price > amount).ToList();
+                    priceFiltered = items.Where(n => n.Price > amount).ToList();
```

Comment:
the `switch` could be done inside the lambda maybe, then you don't have to repeat `items.Where` all the time